docs/tutorials/tutorials.md

```diff
@@ -0,0 +1,382 @@
+    </group>
+    ```
+#### NeXus Field and Attribute
+A NeXus `group` may contain NeXus `fields`, NeXus `attributes`, and other NeXus `groups`. A `field`, which is written as a string without a preceding `NX`, and an `attribute`, preceded by `\@`, must have a [NeXus type](https://manual.nexusformat.org/nxdl-types.html#index-0) (e.g.`NX_FLOAT`, `NX_CHAR`). The NeXus type type must be denoted inside parenthesis (e.g. `end_time(NX_DATE_TIME)`); if the type is ommited, the NeXus `field` or `attribute` has a implicit type `NX_CHAR` by default. Other XML attributes or properties of the NeXus `field`/`attribute`/`group`/`doc` can be defined using one of the special keywords (see `Special Keywords in YAML` below). The descriptive text for NeXus `field`/`attribute`/`group`/`link` is given within the `doc` child.
+
+**NeXus field and attribute in YAML and XML format**
+
+=== "YAML"
+    ```yaml
+    (NXentry):
+      definition:  # Field type: NX_CHAR
+        \@version:  # Attribute type: NX_CHAR
+        enumeration: [NXmpes]
+      title:
+      start_time(NX_DATE_TIME):  # Field type: NX_DATE_TIME
+        doc: Datetime of the start of the measurement.
+      end_time(NX_DATE_TIME):  # Field type: NX_DATE_TIME
+        exists: recommended
+        doc: Datetime of the end of the measurement.
+    ```
+=== "XML"
+    ```xml
+    <group type="NXentry">
+      <field name="definition" type="NX_CHAR">
+        <attribute name="\@version"/>
+        <enumeration>
+          <item value="NXmpes"/>
+        </enumeration>
+      </field>
+      <field name="title"/>
+      <field name="start_time" type="NX_DATE_TIME">
+        <doc>Datetime of the start of the measurement.</doc>
+      </field>
+      <field name="end_time" type="NX_DATE_TIME" recommended="true">
+        <doc>Datetime of the end of the measurement.</doc>
+      </field>
+    </group>
+    ```
+
+#### NeXus Link
+The NeXus `link` reduces data duplication since several concepts of the same kind (e.g., NeXus `group`, `field`, or `attribute`) can refer to a single copy of a data element. In YAML format, NeXus `link` is defined denoting the link inside parenthesis. At the same time, the target concept containing the data must be mentioned under the `target` child.
+
+
+
+**NeXus link in YAML and XML format**
+
+=== "YAML"
+    ```yaml
+    reference_measurement(link):
+      target: /entry
+      doc: A link to a full data collection.
+    ```
+=== "XML"
+    ```xml
+    <link type="NXentry" target="/entry">
+      <doc>A link to a full data collection.</doc>
+    </link>
+    ```
+
+In the YAML example above, `reference_measurement` is defined as a link refering to an instance of the `NXentry` group with its `target` attribute denoting a value `/entry`. This structure ensures that the concept referencing the data is effectively linked to the designated target, thereby reducing the redundancy and maintaining data integrity within the NeXus framework.
+
+
+<!-- ### NeXus Choice
```

Comment:
We should at least mention that we do not consistently support this

Reply:
addressed!